examples/03-Maxwell/Maxwell2D_Electrostatic.py

```diff
@@ -0,0 +1,196 @@
+# Create rectangle and a circle and assign the material read from the .xlsx file.
+# Create two new polylines and a region.
+
+rect = mod2D.create_rectangle(position=['r_x0', 'r_y0', 'r_z0'],
+                              dimension_list=['r_dx', 'r_dy', 0],
+                              name='Ground', matname=mats[0])
+rect.color = (0, 0, 255)  # rgb
+rect.solve_inside = False
+
+circle = mod2D.create_circle(position=['circle_x0', 'circle_y0', 'circle_z0'], radius='circle_radius',
+                             num_sides='0', is_covered=True, name='Electrode', matname=mats[0])
+circle.color = (0, 0, 255)  # rgb
+circle.solve_inside = False
+
+poly1_points = [[-9, 2, 0], [-4, 2, 0], [2, -2, 0],[8, 2, 0]]
+poly2_points = [[-9, 0, 0], [9, 0, 0]]
+poly1_id = mod2D.create_polyline(position_list=poly1_points,segment_type='Spline', name='Poly1')
+poly2_id = mod2D.create_polyline(position_list=poly2_points, name='Poly2')
+mod2D.split([poly1_id, poly2_id], 'YZ', sides='NegativeOnly')
+mod2D.create_region([20, 100, 20, 100])
+
+##################################################################################
+# Define excitations
+# ~~~~~~~~~~~~~~~~~~
+# Assign voltage excitations to rectangle and circle.
+
+M2D.assign_voltage(rect.id, amplitude=0, name='Ground')
+M2D.assign_voltage(circle.id, amplitude=50e6, name='50kV')
+
+##################################################################################
+# Create initial mesh settings
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Assign a surface mesh to the rectangle.
+
+M2D.mesh.assign_surface_mesh_manual(names=['Ground'], surf_dev=0.001)
+
+##################################################################################
+# Create, validate and analyze the setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create, update, validate and analyze the setup.
+
+setup = M2D.create_setup(setupname=sName)
+setup.props['PercentError'] = 0.5
+setup.update()
+M2D.validate_simple()
+M2D.analyze_setup(sName)
+
+##################################################################################
+# Evaluate the E Field tangential component
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Evaluate the E Field tangential component along the given polylines.
+# Add these operations to the Named Expression list in Field Calculator.
+
+fields = M2D.odesign.GetModule('FieldsReporter')
+fields.CalcStack("clear")
+fields.EnterQty("E")
+fields.EnterEdge("Poly1")
+fields.CalcOp("Tangent")
+fields.CalcOp("Dot")
+fields.AddNamedExpression("e_tan_poly1", "Fields")
+fields.EnterQty("E")
+fields.EnterEdge("Poly2")
+fields.CalcOp("Tangent")
+fields.CalcOp("Dot")
+fields.AddNamedExpression("e_tan_poly2", "Fields")
+
+##################################################################################
+# Create Field Line Traces Plot
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create Field Line Traces Plot specifying as seeding faces
+# the ground, the electrode and the region
+# and as ``In surface objects`` only the region.
```

Comment:
```suggestion
# Create a plot of field line traces
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
# Create a plot of field line traces specifying as seeding faces
# the ground, the electrode and the region
# and as ``In surface objects`` only the region.
```
This isn't clear to me. Are all three things listed seeding faces? Can this perhaps be rewritten more clearly?